server/src/test/testApi.js

```diff
@@ -235,12 +269,12 @@ describe('HelloBooks', () => {
   });
  });
 
- //  after((done) => {
- //   //     User.drop();
- //   //     Books.drop();
- //   sequelize.sequelize.sync({ force: true });
+ // after((done) => {
```

Comment:
remove comment from codebase

Reply:
This file will be removed as it is not doing anything